frontend/src/css/NotificationWall.styl

```diff
@@ -68,13 +68,17 @@
           overflow hidden
           font-weight bold
       &__circle
+        font-size wallCircleSize
+        transition 300ms font-size color
+        color lightBlue
+        margin -20px -10px
```

Comment:
This is a little weird for me, there is not another way to do it?

Reply:
I based the developpment on [this answer](https://stackoverflow.com/a/20327676), I don't know if there is another way to do it without changing the display.

Without negative margins we got a result like:
 (The second notification doesn't have the negative margins)
![image](https://user-images.githubusercontent.com/28917197/150795725-f1d47b0e-97c0-4b3c-9079-0bac1378c63c.png)

Reply:
I'm ok with it, the comment was just to know if you look for it or if it was a hack